lib/oembed/scribehow_fallback_provider.rb

```diff
@@ -0,0 +1,11 @@
+module Oembed
+  class ScribehowFallbackProvider < BaseFallbackProvider
+    def self.domains
+      [/scribehow\.com/]
+    end
+
+    def self.paths
+      [%r{^/embed/.*}]
```

Comment:
The instructions for the embed content block editor says: _just copy & paste the full URL to the page that contains the resource that you'd like to embed_. This means that we expect authors to try to embed the `/shared/` link, which will then be rejected by this check here.

You'll need to accept the `/shared/:id` URL _in addition_ to the embed URL, and you'll want to make sure that this class sets a `resource_url` method that replaces `/shared/` with `/embed/` so that embedding actually works.

Reply:
@yash-learner We should leave setting of additional options to the author - they should be part of the URL anyway, and I think we shouldn't touch those settings.